modules/tide_jira/src/Form/TideJiraConfigForm.php

```diff
@@ -69,6 +69,47 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#title' => $this->t('Issue Type'),
     ];
 
+    $form['field_mappings'] = [
+      '#type' => 'fieldset',
+      '#title' => $this->t('Jira field mappings'),
+    ];
+
+    $form['field_mappings']['content_type'] = [
+      '#type' => 'textfield',
+      '#default_value' => $config->get('content_type'),
+      '#title' => $this->t('Content Type Field ID'),
+    ];
+
+    $form['field_mappings']['node_id'] = [
+      '#type' => 'textfield',
+      '#default_value' => $config->get('node_id'),
+      '#title' => $this->t('Node ID Field ID'),
+    ];
+
+    $form['field_mappings']['site'] = [
+      '#type' => 'textfield',
+      '#default_value' => $config->get('site'),
+      '#title' => $this->t('Site Field ID'),
+    ];
+
+    $form['field_mappings']['site_section'] = [
+      '#type' => 'textfield',
+      '#default_value' => $config->get('site_section'),
+      '#title' => $this->t('Site Section Field ID'),
+    ];
+
+    $form['field_mappings']['page_department'] = [
+      '#type' => 'textfield',
+      '#default_value' => $config->get('page_department'),
```

Comment:
@FleetAdmiralButter what happens if it can not get the department value, then what get sets into the default value. e.g department field is only on content-vic and what if other projects using the tide_jira and doesn't have the department field.

Reply:
Hey @MdNadimHossain, this is the config form for mapping custom fields in Jira - it's not looking at the fields in Drupal. 

It's setting the page_department and editor_department values defined here: https://github.com/dpc-sdp/tide_core/blob/feature/SDP2-77-tide-jira-migration/modules/tide_jira/config/install/tide_jira.settings.yml. These will be set as the default upon installation as the field IDs are shared across projects in a Jira instance.